### ISSUE TITLE
Unable to Reproduce AWS Batch Example

### ISSUE THREAD

**Issue by gmolto** [grycap/scar#283]
Attempting to reproduce the example in https://scar.readthedocs.io/en/latest/batch.html results in the following error when initializing the function:

```
Packing udocker files
Creating function package
Function 'scar-mrbayes-batch' successfully created.
Log group '/aws/lambda/scar-mrbayes-batch' successfully created.
Folder 'scar-mrbayes-batch/input/' created in bucket 'scar-mrbayes'
There was an unmanaged exception in init
```
The log in $HOME/.scar/logs is:

```
KeyError: 'comp_type'
    'type': self.batch_properties['comp_type'],
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/batchfunction.py", line 122, in get_compute_env_args
    creation_args = self.get_compute_env_args()
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/batchfunction.py", line 153, in create_compute_environment
    self.batch.create_compute_environment()
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/providers/aws/controller.py", line 106, in init
    return func(*args, **kwargs)
  File "/Users/gmolto/Documents/GitHub/grycap/scar/src/exceptions.py", line 29, in wrapper
Traceback (most recent call last):
```

**Comment by alpegon** [comment]
There was an error in the creation of the configuration file. I fixed it in PR #285.
Please delete your old configuration file and create a new one running scar again without a config file, so the default one can be generated.
Also I updated the [batch documentation](https://scar.readthedocs.io/en/latest/batch.html) to show what variables need to be defined for being able to use the batch environment.

**Comment by gmolto** [comment]
Solved. Thanks!
